### PR TITLE
build(docs-infra): update in-memory-we-api and karma-jasmine-html-reporter version

### LINKED ISSUE #29802: Schematics for library project uses angular-in-memory-web-api from brandons repo
# 📚 Docs or angular.io bug report

### Description

 A clear and concise description of the problem...

After you download the Library schematic project from https://angular.io/guide/schematics-for-libraries, the package.json file contains below dependency

"angular-in-memory-web-api": "github:brandonroberts/in-memory-web-api-bazel#50a34d8",

aio/tools/examples/shared/package.json File path.
Now in most of the organization the only way we can download is via artifactory and we get error. And we are not even using this package in our example.

### What's the affected URL?**
 https://angular.io/...

https://angular.io/guide/schematics-for-libraries

### Expected vs Actual Behavior**

npm install should work behind proxy as well, which is breaking due to the package mentioned.

## 🔥 Exception or Error
<pre><code>

</code></pre>

### PR DESCRIPTION
**Pull request by santoshyadavdev**
Fixes #29802

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #29802

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by gkalpak — APPROVED** [review]
Thx, @santoshyadav198613 :+1:
Can you please update the commit message to `build(docs-infra): ...`.

Also, (although not directly related to this PR) I noticed that `npm test` on examples fails due to [this issue with `karma-jasmine-html-reporter`][1]. If you could update to the latest `karma-jasmine-html-reporter` too (in this PR or another one), that would be rad (but is not blocking this PR) 🔥

[1]: https://github.com/taras42/karma-jasmine-html-reporter/issues/31

**Comment by santoshyadav198613** [comment]
> Thx, @santoshyadav198613 
> Can you please update the commit message to `build(docs-infra): ...`.
> 
> Also, (although not directly related to this PR) I noticed that `npm test` on examples fails due to [this issue with `karma-jasmine-html-reporter`](https://github.com/taras42/karma-jasmine-html-reporter/issues/31). If you could update to the latest `karma-jasmine-html-reporter` too (in this PR or another one), that would be rad (but is not blocking this PR)

Will do that captain :)

**Comment by santoshyadav198613** [comment]
Done with the changes.

File: aio/tools/examples/shared/package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "karma-chrome-launcher": "~2.2.0",
     "karma-coverage-istanbul-reporter": "~2.0.1",
     "karma-jasmine": "~2.0.1",
-    "karma-jasmine-html-reporter": "^0.2.2",
+    "karma-jasmine-html-reporter": "^1.4.2",
```
Comment (gkalpak) [inline aio/tools/examples/shared/package.json]: Thx for making the change. Can you, please, also update the commit message (to indicate that this is not just about `in-memory-web-api` (or move this change to a separate PR) and also mention that this fixes taras42/karma-jasmine-html-reporter#31?

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/tools/examples/shared/package.json (hunk 1 above)
Comment (gkalpak) [reply aio/tools/examples/shared/package.json]: Thx, can you please mention that the `karma-jasmine-html-reporter` fixes lso mention that this fixes taras42/karma-jasmine-html-reporter#31 in the commit message?

Comment (santoshyadav198613) [reply aio/tools/examples/shared/package.json]: Let me know if its fine I was not able to mention that in the message so added in Fixes

**Comment by santoshyadav198613** [comment]
Hi @gkalpak ,
Let me know if any changes are needed,

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
Thx for the ping, @santoshyadav198613. I updated the commit message to be more accurate and marked this for merging.

**Comment by santoshyadav198613** [comment]
Hi @gkalpak ,
Ci failed with  error Command "build-with-ivy-ci" not found. Should i sync the branch?

**Comment by gkalpak** [comment]
Yes, please ❤️

**Comment by santoshyadav198613** [comment]
> Yes, please

Hi @gkalpak ,
Done, and CI is green now.
